app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx

```diff
@@ -136,10 +136,20 @@ export class LogItemInfoTabs extends Component {
       : null;
   }
 
-  componentDidUpdate() {
-    if (this.props.loading && this.props.isSauceLabsIntegrationView) {
+  componentDidUpdate(prevProps) {
+    const {
+      activeTabId,
+      fetchFirstAttachments,
+      isSauceLabsIntegrationView,
+      loading,
+      logId,
+    } = this.props;
+    if (loading && isSauceLabsIntegrationView) {
       this.props.onToggleSauceLabsIntegrationView();
     }
+    if (prevProps.logId !== logId && activeTabId === ATTACHMENTS_TAB_ID) {
+      fetchFirstAttachments();
+    }
```

Comment:
It seems that attachments are not updated when navigating along the arrows.
Please recheck your solution.